tailwind.config.js

```diff
@@ -44,6 +44,8 @@ export default {
         danger20: '#3E0000',
         danger10: '#3E0000',
       },
+
+      fontFamily: { body: [`Roboto`, `Azeret Mono`] },
```

Comment:
We need to take more controll on the fonts. Your option will set the fonts for all html documents, but if user has some troubles with them and these fonts will not be rendered in user's browser? For this reason, you should insert fontFamily like below, because monospace and sans-serif are supported by all browsers and will be rendered anyway.

fontFamily: {
      azeret: ['"Azeret Mono"', 'monospace'],
      roboto: ['Roboto', 'sans-serif'
}